designs/cloudWatch/cloudWatchIntegration.md

```diff
@@ -0,0 +1,62 @@
+# CloudWatch Logs Experience
+
+* **Type**: UX Design
+* **Status**: Proposal
+
+## Introduction
+CloudWatch Logs enables a user to centralize the logs from all of their systems, applications, and AWS services that they use, in a single, highly scalable service. This document describes adding CloudWatch Logs into the JetBrains toolkit in order to facilitate easily viewing and searching them for specific error codes or patterns.
+
+## Comparison to other products
+
+| Product                       | Difference    |
+|-------------------------------|---------------|
+| AWS Toolkit for Visual Studio | Not supported |
+| AWS Toolkit for VSCode        | Not supported |
+
+## Experience
+
+### View Log Groups
+
+#### Alternative
+Instead of building a new CloudWatch service node in the AWS Explorer, we instead build a CloudWatch Log Group browser tab in the CloudWatch tool window. This new tab would also be linked to the active AWS connection settings. This would differ from the other tabs which may lead to a confusing user experience and seen as a con.
```

Comment:
I'm worried about the discoverability of the tab vs an explorer node. Agreed that it makes more sense from a UX perspective, especially since most accounts have a ton of groups and streams, but maybe there should be a node that opens it or something?

Reply:
Ya, I honestly do not know.... I hope the team can give their ideas on this point

Reply:
I think it has to be a log-group list in the explorer as the entry-point.